### DISCUSSION TITLE
Scalability assurance ahead of big traffic

### DISCUSSION THREAD

**Discussion by deHB6** [vercel/community#4643] [Feature Question]
### Summary

We will be launching our application to the public soon and expect significant traffic.  We want to ensure there are no scalability hiccups causing lack of availability.  We are aware of an incident that Arbitrum's website https://arbitrum.io/ suffered earlier this year during their airdrop due to exceeding the default limit of 10,000 concurrent edge functions running and want to ensure we get those settings changed in advance to avoid a similar issue. https://vercel.com/guides/what-should-i-do-if-i-receive-a-429-error-on-vercel

What settings are the biggest culprits and how can we get them changed? 

### Example

https://vercel.com/guides/what-should-i-do-if-i-receive-a-429-error-on-vercel

### Steps to Reproduce

see link with example to what happened to arbitrum

**Comment by amyegan** [comment]
Hi @deHB6. You can find more info in our [Limits](https://vercel.com/docs/limits/overview#), [Terms of Service](https://vercel.com/legal/terms), and [SLA](https://vercel.com/legal/sla) documents. 

If your team is on an Enterprise plan, please coordinate with your Customer Success Manager to ensure a smooth launch! 😄
